pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -60,6 +60,12 @@ type GCECompute interface {
 	GetSnapshot(ctx context.Context, snapshotName string) (*compute.Snapshot, error)
 	CreateSnapshot(ctx context.Context, volKey *meta.Key, snapshotName string) (*compute.Snapshot, error)
 	DeleteSnapshot(ctx context.Context, snapshotName string) error
+	// Metadata
+	GetProject() string
```

Comment:
This breaks the abstraction, the project comes from the metadata service not the Compute service